### DISCUSSION TITLE
milvus 9091 未授权访问

### DISCUSSION THREAD

**Discussion by sly123197811** [milvus-io/milvus#32305] [Q&A and General discussion]
Version: milvus2.3
The company has detected the following security vulnerability in port 9091 of the milvus service: Redis Exporter metrics unauthorized access
I have studied the official website and configuration files, but did not see where Redis Exporter is used in Milvus. How can I modify the Redis configuration in Milvus

**Comment by yhmo** [comment]
9091 is a http port of Milvus to output runtime metrics. From the source code, I didn't find any dependency on Redis.
